v20.2/build-a-typescript-app-with-cockroachdb.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Build a TypeScript App with CockroachDB and TypeORM
+summary: Learn how to use CockroachDB with the TypeORM framework.
+toc: true
+twitter: false
+---
+
+This tutorial shows you how run a simple application built with [TypeORM](https://typeorm.io/#/).
+
+## Step 1. Start CockroachDB
+
+{% include {{page.version.version}}/app/start-cockroachdb.md %}
+
+## Step 2. Create a database
+
+{% include {{page.version.version}}/app/create-a-database.md %}
+
+### Step 3. Get the code
+
+Clone [the code's GitHub repository](https://github.com/typeorm/cockroachdb-example).
+
+### Step 4. Update the connection parameters
+
+Open the `ormconfig.ts` file, and edit the ORM configuration parameters:
+
+<section class="filter-content" markdown="1" data-scope="local">
+
+- Replace the value for `port` with the port to your cluster.
+- Replace the value for `username` with the user you created earlier.
+- Replace the value for `password` with the password you created for your user.
+
+</section>
+
+<section class="filter-content" markdown="1" data-scope="cockroachcloud">
+
+- At the top of the file, uncomment the `import * as fs from "fs";` line.
+
+    This line imports the `fs` Node module, which enables you to read in the CA cert that you downloaded from the CockroachCloud Console.
+- Replace the value for `host` with the name of the CockroachCloud Free host (e.g., `host: 'free-tier.gcp-us-central1.cockroachlabs.cloud'`).
+- Replace the value for `port` with the port to your cluster.
+- Replace the value for `username` with the user you created earlier.
+- Replace the value for `password` with the password you created for your user.
+- Replace the value for `database` with the database that you created earlier, suffixed with the name of the cluster (e.g., `database: '{cluster_name}.bank'`).
+- Remove the `ssl: true` key-value pair.
+- Remove the `extra` object and its contents.
+- Uncomment the `ssl` object with the `ca` key-value pair, and edit the `fs.readFileSync('certs/cc-ca.crt').toString()` call to use the path to the `cc-ca.crt` file that you downloaded from the CockroachCloud Console.
```

Comment:
These settings aren't in `ormconfig.json` either. I feel like maybe you're referring to a version of the app that isn't in https://github.com/typeorm/cockroachdb-example

Reply:
:face_palm:

I was linking to the wrong repo. My bad.

 I had to write my own app, because https://github.com/typeorm/cockroachdb-example wasn't working.

The app I made is here: https://github.com/cockroachlabs/hello-world-typescript-typeorm